client/lib/invites/reducers/invites-validation.js

```diff
@@ -2,22 +2,39 @@
  * External dependencies
  */
 import { fromJS } from 'immutable';
+import mapValues from 'lodash/object/mapValues';
+import pick from 'lodash/object/pick';
 
 /**
  * Internal dependencies
  */
 import { action as ActionTypes } from 'lib/invites/constants';
+import { decodeEntities } from 'lib/formatting';
 
 const initialState = fromJS( {
 	list: {},
 	errors: {}
 } );
 
+function filterObject( object ) {
```

Comment:
could we change the name of this function for something more descriptive? something like `decodeProperties` or `decodeObjectProperties` or similar? with `filterObject` you don't really get what it does unless you look at the code of the function